experiments/kompanion/cmd/export/export.go

```diff
@@ -90,10 +95,110 @@ func init() {
 
 	ExportCmd.Flags().StringArrayVarP(&targetObjects, targetObjectsFlag, "", []string{}, "object name prefix to target. Targets all if empty. Can be specified multiple times.")
 	ExportCmd.Flags().StringArrayVarP(&ignoreObjects, ignoreObjectsFlag, "", []string{}, "object name prefix to ignore. Excludes nothing if empty. Can be specified multiple times.")
+
+	ExportCmd.Flags().IntVarP(&workerRountines, workerRoutinesFlag, "", 10, "Configure the number of worker rountines to export namespaces with. Defaults to 10. ")
```

Comment:
Nit: I don't think there's any reason to define a constant here.

Reply:
Should we assume no parallelism in the case where the flag is not set?

Reply:
I'd go with 1000 :-)

Reply:
Even with 10 already seeing a lot of :
I0802 16:28:52.334712 2829986 request.go:697] Waited for 1.156100829s due to client-side throttling, not priority and fairness, request: GET:https://34.134.156.190/apis/accesscontextmanager.cnrm.cloud.google.com/v1beta1/namespaces/gke-managed-system/accesscontextmanageraccesspolicies